drivers/driver.go

```diff
@@ -36,8 +25,8 @@ type RDMAStorageController interface {
 }
 
 // NewRDMAVolumeDriver constructs a new RDMAVolumeDriver.
-func NewRDMAVolumeDriver(sc RDMAStorageController) RDMAVolumeDriver {
-	return RDMAVolumeDriver{sc}
+func NewRDMAVolumeDriver(sc RDMAStorageController, vd db.VolumeDatabase) RDMAVolumeDriver {
```

Comment:
Short variable names hinder the reader's ability to understand this code. Please lengthen them for all our sakes.